src/shared/js/pie.js

```diff
@@ -87,9 +87,18 @@ export default class Pie {
   set weight(weight) {
     this._attrs.weight = isNaN(weight) ? this._attrs.weight : weight;
   }
+  set serves(serves) {
+    this._attrs.serves = isNaN(serves) ? this._attrs.serves : serves;
```

Comment:
Sorry I didn't notice this earlier, but I'm not sure if this check is sufficient. Can `serves` ever be `null` or an empty string value, for example? If so, `isNaN` will still return `false`, even though they're not useful numbers. Perhaps we should have e.g. `isNaN(parseInt(serves, 10))`?

Sorry if I'm missing a step that makes this all work, but it's worth pointing out in case I didn't get it wrong :)

Reply:
You want to be able to null it, to remove the value (serves 0 is different than serves not set). Theoretically we could remove object prop in this case, but I think it is easier to set to null in a json-driven db like fb. Wdyt?

Reply:
That sounds reasonable, yes. `isNaN` should at least catch nonsense values like "banana", and we can always refine later to e.g. complain about negative numbers.